enterprise/server/workflow/service/service.go

```diff
@@ -694,7 +708,11 @@ func (ws *workflowService) createActionForWorkflow(ctx context.Context, wf *tabl
 	return actionDigest, err
 }
 
-func (ws *workflowService) workflowsPoolName() string {
+func (ws *workflowService) workflowsPoolName(os string) string {
+	// When using firecracker, don't use the workflows pool.
+	if *enableFirecracker && (os == "" || os == platform.LinuxOperatingSystemName) {
+		return platform.DefaultPoolValue
```

Comment:
I think it's still useful to have a separate pool for workflows+hosted bazel due to auto-scaling on the main pool. 
Any workflow or hosted bazel instances that ends up on a VM created after a scale-up will be lost if the pool scales down and will have to go through a cold start.

Reply:
Reverted this change